market/views.py

```
@@ -14,6 +14,11 @@
 from django.http import HttpResponseRedirect, JsonResponse
 from django.urls import reverse
 from decimal import Decimal
+from django.contrib.auth.forms import PasswordResetForm
```

Ruff (0.12.2)

`django.contrib.auth.forms.PasswordResetForm` imported but unused

Remove unused import: `django.contrib.auth.forms.PasswordResetForm`

(F401)